app/src/main/kotlin/com/twain/interprep/presentation/ui/components/resource/ResourceCard.kt

```diff
@@ -70,14 +74,14 @@ fun ResourceCard(
                     text = resource.topic,
                     style = MaterialTheme.typography.bodyLarge
                 )
-                if (!resource.subtopic.isNullOrEmpty()) {
+                if (resource.subtopic.isNotEmpty()) {
                     Text(
                         text = resource.subtopic,
                         style = MaterialTheme.typography.bodyMedium
                     )
                 }
                 if (isExpanded) {
-                    resource.links.forEachIndexed { index, link ->
```

Comment:
There is one bug in the UI where the edit pen is hidden when the link list is expended. I didn't find the solution yet